src/clnt_dg.c

```diff
@@ -146,6 +146,11 @@ clnt_dg_ncreatef(const int fd,	/* open file descriptor */
 	}
 	su = su_data(xprt);
 
+	/*
+	 * Take extra ref for rpc client.
+	 */
+	SVC_REF(xprt, SVC_REF_FLAG_NONE);
```

Comment:
Where is this extra ref released?

Reply:
> Where is this extra ref released?

On connection reset svc_vc_recv(SVC_DESTROY) and svc_rqst_xprt_task(SVC_RELEASE) will release 2 refs, so we just need to release 1 ref in clnt_destroy.
If we are in clnt_destroy without any connection reset then we will be having 3 refs, 1st ref should be released by SVC_RELEASE, 2nd ref should be released by SVC_DESTROY. So we need one more SVC_RELEASE here.